.github/workflows/publish-docker-image.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish multi-arch Docker Image
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Login to Docker Hub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+    - name: Docker info
+      run: docker info
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+    - name: Buildx inspect
+      run: docker buildx inspect
+    - name: Build and push
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./Dockerfile
+        platforms: linux/amd64,linux/arm/v7,linux/arm64,linux/ppc64le,linux/s390x
+        push: true
+        tags: |
+          testcontainers/ryuk:latest
```

Comment:
I would suggest we push only the tag and not `:latest`, to avoid a confusion when there are more changes on `master` but `:latest` still points to the "latest release", but previously our `:latest` was more like `-SNAPSHOT`, on every `master` commit

Reply:
Sure, I'm happy not to update `latest` - we should discourage its use.